Sources/ProjectDescription/FileList.swift

```diff
@@ -2,28 +2,47 @@ import Foundation
 
 public struct FileList: Codable, Equatable {
     /// List glob patterns.
-    public let globs: [Path]
+    public let globs: [HeaderFileGlob]
 
     /// Initializes the files list with the glob patterns.
     ///
-    /// - Parameter globs: Glob patterns.
-    public init(globs: [Path]) {
+    ///   - glob: Relative glob pattern.
+    ///   - excluding: Relative glob patterns for excluded files.
+    public init(_ globs: [HeaderFileGlob]) {
         self.globs = globs
     }
+    
+    // for backward compatibility when globs property had [Path] type
+    public init(globs: [Path]) {
```

Comment:
Since you are touching this file, I'd recommend adding a `.headers` static method. In the next major version, we'll make those `init` private.

Reply:
@pepicrft but maybe ".list" instead of ".headers" as we decided not make these classes limit to headers only

Reply:
done

Reply:
I still see the only the pubblic `init` 🤔

Reply:
I can't change this as this method was before changes (otherwise we will break API in the not major version)